### DISCUSSION TITLE
Add a content editor

### DISCUSSION THREAD

**Discussion by RDjarbeng** [strapi/strapi#8743] [Legacy Topics]
Does anyone know how to add a user solely for editing the content that has been created on strapi? I followed the tutorial for deploying gatsby with strapi. I would like to add another user without admin privileges. I would be grateful for any links provided

**Comment by RDjarbeng** [comment]
Well I guess I'll answer the question myself

Using strapi v3.3.3 Community Edition
1. Under 
> Settings> Administration Panel>Users

 You can add a user by clicking 'create new user' at the top right and selecting role as editor if you want the user to be able to edit and view content posted by other users. Or Author for the user to edit content for which he is the author. Fill in the necessary information and continue.
**Do not use the User given under collection types**
_That was my mistake. That user collection apparently is not used for adding content editors._

2. A link will be generated for the user to use to verify and login with their details (password ). Copy the link. Read the next step before leaving.

3. Then under Roles>editor or Roles>author depending on which you want select the collection types for which you want the user to be able to access. If you do not enable access for a collection type the user will see no collections to edit when they login.

4. Open a new tab and let the user login with the given link in step 2 or you can log in with the details since this is a test. On a local machine it will be similar to:
`http://localhost:1337/auth...`

5. All done. The user can now edit and change content without having access to the content builder.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
